.chloggen/expose_more_protocols.yaml

```diff
@@ -0,0 +1,19 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. operator, github action)
+component: operator
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Expose supported protocols on the distributor
+
+# One or more tracking issues related to the change
+issues: [436]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: |
+  The following protocols are now exposed: 
+    - Jaeger: thrift-http thrift-binary, thrift-compact, Grpc
```

Comment:
> > Is this ok to be merged?
> 
> I didn't merge it yet because @pavolloffay asked for port numbers in the changelog entry - not sure if this is a blocker or not.

Done, added ports.